src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -67,7 +67,6 @@ public EditCommand parse(String args) throws ParseException {
             editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()));
         }
         parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).ifPresent(editPersonDescriptor::setTags);
-        parseLogsForEdit(argMultimap.getAllValues(PREFIX_LOG)).ifPresent(editPersonDescriptor::setLogs);
```

Comment:
It seems like the user can still enter the logs field during edit, and nothing would happen. Would it be better if we could throw an error to tell the users that editing of logs is not allowed?

Reply:
yup make sense! will work on it